Abbyy.CloudSdk.V2.Client.Sample/Program.cs

```diff
@@ -33,36 +38,92 @@ public class Program
 		/// </summary>
 		private const string ServiceUrl = "https://cloud-eu.ocrsdk.com";
 
-		private static AuthInfo _authInfo;
+		private static int _retryCount = 3;
+		private static int _delayBetweenRetriesInSeconds = 3;
+		private static string _httpClientName = "OCR_HTTP_CLIENT";
+
+		private static readonly AuthInfo _authInfo = new AuthInfo
+		{
+			Host = ServiceUrl,
+			ApplicationId = ApplicationId,
+			Password = Password
+		};
+
+		private static ServiceProvider _serviceProvider;
+		private static HttpClient _httpClient;
 
 		public static async Task Main()
 		{
-			_authInfo = new AuthInfo
-			{
-				Host = ServiceUrl,
-				ApplicationId = ApplicationId,
-				Password = Password,
-			};
+			// Create service collection and configure our services
+			var services = ConfigureServices();
+			// Generate a provider
+			_serviceProvider = services.BuildServiceProvider();
 
-			var resultUrls = await ProcessImageAsync();
+			var httpClientFactory = _serviceProvider.GetService<IHttpClientFactory>();
+			_httpClient = httpClientFactory.CreateClient(_httpClientName);
 
-			foreach (var resultUrl in resultUrls)
-			{
-				Console.WriteLine(resultUrl);
-			}
+			//Run examples
```

Comment:
Why is this code turned off here and turned on in core sample?

Reply:
it's debugging code, i'll fix it